javaOOP_workingWithAbstraction/src/jv/oop/working/with/abstraction/lab/rhombus/stars/RhombusOfStars.java

```diff
@@ -0,0 +1,48 @@
+package jv.oop.working.with.abstraction.lab.rhombus.stars;
+
+import java.util.Scanner;
+
+/**
+ * Create a program that reads a positive integer n as input and prints on the
+ * console a rhombus with size n:
+ * 
+ * @author Ivelin Gavrailov
+ * @since 04.01.2021
+ *
+ */
+public class RhombusOfStars {
+
+	public static void main(String[] args) {
+		Scanner scan = new Scanner(System.in);
+		int n = Integer.parseInt(scan.nextLine());
+		printRhombus(n);
+		scan.close();
+	}
+
+	private static void printRhombus(int rows) {
```

Comment:
Good job to separate the parts of the method in different methods.

Reply:
thank you